src/graph/visitor/FoldConstantExprVisitor.cpp

```diff
@@ -338,6 +338,11 @@ void FoldConstantExprVisitor::visitBinaryExpr(BinaryExpression *expr) {
 }
 
 Expression *FoldConstantExprVisitor::fold(Expression *expr) {
+  // Container expresison should remain the same type after being folded
+  if (expr->isContainerExpr()) {
```

Comment:
Why don't fold container expression? How about `[1 + 1, 2]` ?

Reply:
`[1+1, 2]` are folded into `[2, 2]`. The old implementation converts the container expression to a nebula::value list.